src/LiveStorage.test.ts

```diff
@@ -0,0 +1,8 @@
+import { LiveStorage } from './LiveStorage'
+
+test('REAME example does not throw', () => {
+  // TODO: BroadcastChannel is not available in Node 14 nor `jsdom` environment
+  const posts = new LiveStorage('posts', [])
+
+  posts.update((prevPosts) => prevPosts.concat({ title: 'Brave new world' }))
```

Comment:
It would be nice to have some assertion in this test. Since we don't have a value getter at the moment (#1), we may check that those invocations do not throw by themselves:

```js
const scenario = () => {
  const posts = new LiveStorage(...)
}

expect(scenario).not.toThrow()
```

Reply:
updated with explicit assertion..

technically, Jest was already asserting the not throwing, that's just how Jest works,, but I understand that an explicit assertion might be more readable for a wide audience :)

Reply:
I didn't know that, thanks for sharing. Still, an explicit assertion would be good, as it's not even the final one (in reality we'd implement a value getter and test a set/get scenario to produce a meaningful test).